core/src/components/segment-button/segment-button.ionic.scss

```diff
@@ -0,0 +1,107 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@use "./segment-button.common";
+
+// Ionic Segment Button
+// --------------------------------------------------
+
+:host {
+  /**
+   * @prop --color-disabled: Color of the disabled segment button
+   */
+  --background: #{globals.$ionic-color-base-white};
+  --background-checked: var(--background);
+  --color: #{globals.$ionic-color-neutral-1000};
+  --color-checked: #{globals.$ionic-color-primary-base};
+  --color-disabled: #{globals.$ionic-color-neutral-500};
+  --border-width: #{globals.$ionic-border-size-025};
+  --border-color: #{globals.$ionic-color-neutral-300};
+  --border-style: #{globals.$ionic-border-style-solid};
+  --indicator-box-shadow: none;
+  --indicator-color: var(--color-checked);
+  --indicator-height: #{globals.$ionic-border-size-025};
+  --indicator-transition: transform 250ms cubic-bezier(0.4, 0, 0.2, 1);
+  --indicator-transform: none;
+  --padding-top: #{globals.$ionic-space-200};
```

Comment:
This was done before design made some updates, but they have updated it so that the outer item has padding & the inner content does too. Updated here: https://github.com/ionic-team/ionic-framework/pull/29897/commits/6234434b752589eb2c0f58945fd67c26bc8e046c